src/metadata/persistence/permanent_id_requests.clj

```diff
@@ -198,7 +198,7 @@
 (defn update-permanent-id-request
   "Records the Permanent ID for a given Request."
   [request-id permanent-id]
-  (sql/update :permanent_id_requests
+  (ksql/update :permanent_id_requests
```

Comment:
These updates are the bits in progress. I've been changing things over to use the `ksql` name, since `jsql` I'm using for next.jdbc. Should go away ultimately anyway.